core/block_rendering_rewrite/measurables.js

```diff
@@ -208,6 +263,86 @@ Blockly.BlockRendering.ExternalValueInput = function(input) {
 goog.inherits(Blockly.BlockRendering.ExternalValueInput,
     Blockly.BlockRendering.Input);
 
+/**
+ * An object containing information about the space a previous connection takes
+ * up during rendering.
+ * @package
+ * @constructor
+ */
+Blockly.BlockRendering.PreviousConnection = function() {
+  Blockly.BlockRendering.PreviousConnection.superClass_.constructor.call(this);
+  this.renderRect = null;
```

Comment:
If you're not using `renderRect`, remove it.  On `render/collab` it's only ever set to null, and never used.